shell/app/modules/project/pages/release/components/release-select.tsx

```diff
@@ -12,14 +12,24 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 import React from 'react';
-import { Dropdown, Row, Col, Button, Input } from 'antd';
+import { Dropdown, Row, Col, Button, Input, Timeline, Collapse, Modal } from 'antd';
 import i18n from 'i18n';
 import { ErdaIcon, Pagination } from 'common';
 import { IPaginationProps } from 'common/components/pagination';
 import { useUpdateEffect } from 'react-use';
 import empty from 'app/images/empty.svg';
 
-import './index.scss';
+import './release-select.scss';
+
+const { Panel } = Collapse;
+
+const groupsTitle = [
+  i18n.t('dop:the {index} group', { index: i18n.t('dop:first') }),
+  i18n.t('dop:the {index} group', { index: i18n.t('dop:second') }),
```

Comment:
I suggest these i18n should belong to default ns.
better remove `the` in en. Then it would be 
`{index}组` in Zh
`{index} group` in En
the Zh translation should be `第N`

Reply:
After talking to UI, he agreed

Reply:
Should be `group {index}` instead of `{index} group(s)`